docs/README.md

```diff
@@ -138,6 +138,9 @@ module.exports.handler = serverless(service)
   // - If it returns `null`, no proxy will be used and the default factory will be skipped entirely.
   // Default: the built-in proxy factory from `fast-gateway`
   proxyFactory: ({ proxyType, opts, route }) => {...}
+   // Optional toggle for exposing minimal documentation of registered services at `GET /services.json`
+   // Default value: true
+  servicesJsonRoute: true
```

Comment:
let use `enableServicesEndpoint` as attribute name instead

Reply:
I’ll try to fix this tomorrow, but short on time.